### ISSUE TITLE
BlurView BlurViewManager not found

### ISSUE THREAD

**Issue by kscorpio** [catalinmiron/react-native-dribbble-app#14]
clang: error: no such file or directory: '/Users/liu/Downloads/react-native-dribbble-app-master 2/node_modules/react-native-blur/RNBlur/BlurView.m'
clang: error: no input files

clang: error: no such file or directory: '/Users/liu/Downloads/react-native-dribbble-app-master 2/node_modules/react-native-blur/RNBlur/BlurViewManager.m'
clang: error: no input files

**Comment by seanmthompson** [comment]
+1 Having the same issue as well. Can't build.

**Comment by catalinmiron** [comment]
Can you please provide some screenshots? I was able to build the app without having these errors.

**Comment by seanmthompson** [comment]
![screen shot 2015-09-28 at 9 44 09 am](https://cloud.githubusercontent.com/assets/5447621/10142113/7922a6b0-65c5-11e5-9eaa-ad2d0fcc8f53.png)

**Comment by catalinmiron** [comment]
@seanmthompson can you please remove `node_modules` and `npm install` again? 
Let me know if this issue still persist after reinstalling modules. Thanks

**Comment by seanmthompson** [comment]
Yeah still persisting. Even re-downloaded the source and installed node modules. What's weird is that Xcode says those dependencies don't exist, but they are showing in Finder as installed.

**Comment by seanmthompson** [comment]
Ok isolated the issue. Had to manually add the BlurView dependencies in Xcode. (didn't know I had to do that!)

**Comment by catalinmiron** [comment]
It's really weird, since react-native-blur was in `node_modules`.

> Ok isolated the issue. Had to manually add the BlurView dependencies in Xcode. (didn't know I had to do that!)

This is what I should told you in the first place. I'm glad that you made it :+1: 

Btw, thanks for checking the app!

**Comment by seanmthompson** [comment]
Thanks for the quick feedback! New to React Native (and Xcode) so still learning the ins and outs!

**Comment by ravalbhavin07** [comment]
how to manually add and which files? , want to get this working in xcode to see the dribble example.
